backend/package.json

```diff
@@ -5,10 +5,10 @@
   "scripts": {
     "start": "node ./bin/www",
```

Comment:
Replace this with this code
`    "start": "node app.js",
    "start:dev": "nodemon ./bin/www",
    "backend": "nodemon app.js"
`
we can run nodemon with this command now "npm run backend"